src/rust/engine/src/intrinsics.rs

```diff
@@ -129,15 +130,35 @@ fn multi_platform_process_request_to_process_result(
         str
       ))
     })?;
-    let result = context.get(process_request).await?;
-    let platform_name: String = result.0.platform.into();
+    let result = context.get(process_request).await?.0;
+
+    let maybe_stdout = store
+      .load_file_bytes_with(result.stdout_digest, |bytes: &[u8]| bytes.to_owned())
+      .await
+      .map_err(|s| throw(&s))?;
+
+    let maybe_stderr = store
+      .load_file_bytes_with(result.stderr_digest, |bytes: &[u8]| bytes.to_owned())
+      .await
+      .map_err(|s| throw(&s))?;
+
+    let extract_bytes = |maybe_digest: Option<(Vec<u8>, _)>| {
+      maybe_digest
+        .map(|(bytes, _load_metadata)| bytes)
+        .ok_or_else(|| throw("Bytes from Digest not found in store"))
```

Comment:
If this ever triggered, it would be because of something like #9942. Although (we should fix that so) that should be incredibly rare, we should still include a bit more information here, including 1) which digest we failed to load, 2) what we were trying to load (so maybe take a static reference so that you can mention `stdout` vs `stderr`),